### DISCUSSION TITLE
My Github teacher developer account is suspended for the second time for a baseless reason.

### DISCUSSION THREAD

**Discussion by mridun-gupta** [community/community#17713] [GitHub Education]
I request the Github support team to get it un-suspended, but they got me the reply as shown here:
![image](https://user-images.githubusercontent.com/91019667/169422116-ba88dbef-2f6e-4628-9e99-417c8b753d9a.png)

But replied to the team accordingly explaining to them why there were a bit high requests for Swags, as shown here:
![image](https://user-images.githubusercontent.com/91019667/169422267-bcd08a2f-8579-4607-a970-0d0c22bfdb14.png)
![image](https://user-images.githubusercontent.com/91019667/169422280-d11a6904-5446-41b5-a61a-890ffea1e69a.png)

I request the Github support team to unban my Github Teacher Developer Pack as I guarantee that our institution will not follow this practice anymore, and will just request swags once a month.

**Comment by mridun-gupta** [comment]
> I request the Github support team to get it un-suspended, but they got me the reply as shown here: ![image](https://user-images.githubusercontent.com/91019667/169422116-ba88dbef-2f6e-4628-9e99-417c8b753d9a.png)
> 
> But replied to the team accordingly explaining to them why there were a bit high requests for Swags, as shown here: ![image](https://user-images.githubusercontent.com/91019667/169422267-bcd08a2f-8579-4607-a970-0d0c22bfdb14.png) ![image](https://user-images.githubusercontent.com/91019667/169422280-d11a6904-5446-41b5-a61a-890ffea1e69a.png)
> 
> I request the Github support team to unban my Github Teacher Developer Pack as I guarantee that our institution will not follow this practice anymore, and will just request swags once a month.

@tarebyte @BenEmdon @leereilly

**Comment by ettaboyle** [comment]
Hi @mridun-gupta,

This is a very specific issue that you are facing. Our private support team is the only team who will be able to help with requests on account flags. You will want to follow up with them directly through your open ticket.

I am closing this discussion as this is an account-specific topic. Thank you!
